annoto-campus/xblock-in-video-collaboration
# [SE-5115] merge annoto master

Like last time (https://github.com/annoto-campus/xblock-in-video-collaboration/pull/7), we're bringing the new changes that happened at https://github.com/Annoto/xblock-in-video-collaboration, into this branch, used by Campus.

**Dependencies**: None

**Testing and deployment procedure**:
This is what I plan to do.
- We already saw that the code changes are small. But they need to be tested in practice
- Build PRs to update the olive-secure variables, to point to my branch here, so that Campus stage starts using the new blocks
- (I'll add other changes that are required on Campus stage)
- Deploy Campus stage
- Campus will test it there. They'll need a course using Annoto. See links at [the last PR|https://github.com/annoto-campus/xblock-in-video-collaboration/pull/7], maybe it can still be used
- Ask to merge this PR
- Update the olive-secure settings to point to the new commit (the merge) and use it in production and stage
- Deploy Campus production

**Author notes and concerns**

Some of the changes are about changing positions (left to right). We'll need to test that things still look fine.

**Others**
See [document](https://docs.google.com/document/d/1EiZxvSUaqaSN8Bri7S39f7eXaYQ6Ar69MEuDF6CXR-Y/edit#).

**Reviewers**
- @gabor-boros
- Campus
